app/Http/Controllers/InterOp/UserGroupsController.php

```diff
@@ -39,16 +40,16 @@ public function setDefault()
 
     private function getUserAndGroupModels()
```

Comment:
I'm not sure if this function is still useful or can be
```
private function findGroupOrFail($groupId)
{
    ...
}
```

And in the controller actions
```
$group = $this->...($groupId);
$user = User::findOrFail($userId);
```

Considering both are accessible directly from `update($userId, $groupId)` etc (with only exception `setDefault`).

And maybe even add `byIdOrFail` to `Groups` class itself and throw `ModelNotFoundException` accordingly. Could also be useful for `GroupsController#show`.